api/v1beta1/common.go

```diff
@@ -58,3 +58,9 @@ func validateIBMPowerVSNetwork(network IBMPowerVSResourceReference) (bool, IBMPo
 	}
 	return true, IBMPowerVSResourceReference{}
 }
+
+func defaultIBMVPCMachineSpec(spec *IBMVPCMachineSpec) {
+	if spec.Profile == "" {
+		spec.Profile = "bx2-4x16"
```

Comment:
any specific reason for making this profile as a default value?

Reply:
No specific reason as such, we can always default to above value or some other value maybe with lesser resources.

Reply:
lets merge this as of now, will find a bare minimal resources needed part of a different task and update defaults for both vpc and powervs

Reply:
created a task to track this - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/499